plugins/openshift/commands/bmh-ironic-check-redfish-profile.md

```
@@ -0,0 +1,842 @@
+---
+description: Validate BMC Redfish compliance against OpenStack Ironic profile
+argument-hint: "<bmc-address> <username> <password>"
+---
+
+## Name
+openshift:bmh-ironic-check-redfish-profile
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)